demonstrations/tutorial_rosalin.py

```diff
@@ -0,0 +1,598 @@
+r"""
+Frugal shot optimization with Rosalin
+=====================================
+
+.. meta::
+    :property="og:description": In this demonstration, the Rosalin optimizer
+    	is used to minimize a Hamiltonian via an frugal shot optimization strategy.

```

Comment:
Suggest we try to minimize the amount of additional words in the `property` (e.g., "In this demonstration") and be more direct. Try to picture that you're seeing this text below a search engine query.
```suggestion
    :property="og:description": The Rosalin optimizer uses a measurement-frugal optimization strategy to minimize the number of times a quantum computer is accessed.
```
or something similar